src/pkg/packaging/deb/package.targets

```diff
@@ -165,38 +223,74 @@
     <MakeDir Directories="$(debLayoutPackageRoot)" />
     <MakeDir Directories="$(debLayoutSamples)" />
     <MakeDir Directories="$(debLayoutDocs)" />
-    
+
     <!-- Copy files to debian layout -->
     <ItemGroup>
-        <SHFiles Include="$(InputRoot)/**/*" />
-        <SHManpages Include="$(ManPagesDir)/**/*" />
+      <SHFiles Include="$(InputRoot)/**/*" />
```

Comment:
No, `<InputRoot>` is being set above this in the same Target, so it won't be empty.